### PR TITLE
Big refactor of the community page

### PR DESCRIPTION
**Pull request by kevmoo**
Moved references from Slack -> Gitter

### CODE REVIEW

**Comment by kevmoo** [comment]
I think this flows better – thoughts?

**Comment by kwalrath** [comment]
Could you stage it?

**Comment by kevmoo** [comment]
See https://dart-site-www-commuity-updates.firebaseapp.com/community

Did not use naming convention – will make sure to clean this up 😄

**Comment by kwalrath** [comment]
Tag, @kevmoo, you're it.

Did a bunch of tweaks. I'm not thrilled with the opening paragraph, so feel free to change that.

**Comment by kwalrath** [comment]
Oh, and... staged here:

https://kw-www-dartlang-1.firebaseapp.com/community

File: src/community/index.md [hunk 1]
```diff
@@ -5,67 +5,64 @@ title: "Community and Support"
...
-* [Bugs and feature requests](https://github.com/dart-lang/sdk/issues/new)
+Get answers and connect with Dart developers.
 
 #### Communities
 
-* [StackOverflow](http://stackoverflow.com/tags/dart)
-* [Reddit](https://www.reddit.com/r/dartlang)
+* [StackOverflow](http://stackoverflow.com/tags/dart) -
+  The best place for how-to questions.
+* [Gitter](https://gitter.im/dart-lang) -
```
Comment (kwalrath) [inline src/community/index.md]: Yikes, I think this should be https://gitter.im/dart-lang/home.

@kevmoo, could you change it?

Comment (kevmoo) [inline src/community/index.md]: Done!

**Comment by kevmoo** [comment]
PTAL - redeployed here https://dart-site-www-commuity-updates.firebaseapp.com/community

**Comment by Sfshaza** [comment]
lgtm

**Pull request merged**